packages/x-components/build/rollup.config.ts

```diff
@@ -93,10 +93,13 @@ export const rollupConfig = createRollupOptions({
       mode: [
         'inject',
         (varname: string, id: string) =>
-          // eslint-disable-next-line max-len
-          `import {createInjector} from 'vue-runtime-helpers';const injector=createInjector({});injector('${normalizePath(
-            id
-          )}',{source:${varname}})`
+          `import {createInjector, createInjectorSSR} from 'vue-runtime-helpers';
+           const isBrowser = (function(){try {return this!==global;}catch(e){return true;}})();
```

Comment:
This function is returning `true` when I execute it in a Node.js environment, what do you think of changing it to?
```suggestion
           const isBrowser = (function(){try {return this===window;}catch(e){return false;}})();
```

Reply:
FYI, because we are executing this check in a function, rollup is not able to remove it. whenever it feels the resultant variant is not needed. To fix it you can either not execute it in an iife, or flag the function as pure (no side effects) with the `/*#__PURE__*/`.
![image](https://user-images.githubusercontent.com/68222542/186894600-7fb88833-d347-4745-8542-3b4172578335.png)
![image](https://user-images.githubusercontent.com/68222542/186894626-c4703bac-407e-4be7-ad64-0d00d14785e3.png)

Reply:
Update: apparently, it was returning `true` because I ran the script in a project using ES modules instead of CommonJS, which uses `strict` mode by default causing `this` to behave differently. This is another approach, by @javieri-empathy, which seems to work properly:
```suggestion
           const isBrowser = (function(){return (Object.prototype.toString.call(typeof process!=='undefined'? process:0)!=='[object process]');})();
```